scripts/base/protocols/ftp/main.zeek

```diff
@@ -212,8 +212,9 @@ function ftp_message(c: connection)
 	# values after logging.
 	delete s$mime_type;
 	delete s$file_size;
-	# Same with data channel.
+	# Same with data channel and fuid;
```

Comment:
Remove this comment and change the one above to say something like "The following fields are specific".